cg/cli/upload/nipt/base.py

```diff
@@ -1,19 +1,86 @@
 """ Upload NIPT results via the CLI"""
 
 import logging
+import sys
+import traceback
+from datetime import datetime
+from typing import Optional
 
-from .ftp import ftp
-from .statina import statina
+from cg.cli.upload.utils import suggest_cases_to_upload
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+from cgmodels.cg.constants import Pipeline
+
+from .ftp import ftp, nipt_upload_case
+from .statina import statina, batch
 
 import click
 
 LOG = logging.getLogger(__name__)
 
 
-@click.group()
-def nipt():
+@click.group(invoke_without_command=True)
+@click.option("-c", "--case", "case_id", help="Upload to all apps")
```

Comment:
could we make the cli more intuitive and have case id as required argument, instead of optional option?

Reply:
I believe it needs to be optional, it seems that we have to specify it for the subcommands otherwise. I.e. you have to specify case_id for `cg upload nipt auto` which makes little sense

Reply:
nipt group does not need to have logic or arguments, or invoke other subcommands as default behavior. This now is done in a very round-about way which is difficult to follow and maintain later. Individual commands may have arguments that are required (when uploading each case individually) or none (in case of uploading all).

Reply:
fixed